src/Cli/dotnet/commands/dotnet-sdk/check/Program.cs

```diff
@@ -67,4 +78,12 @@ public static int Run(string[] args)
             return new SdkCheckCommand(parseResult).Execute();
         }
     }
+
+    internal class SdkCheckConfig
```

Comment:
Can we document this format somewhere (as well as the path it needs to be in)?  If nothing else, we could put it in the PR description.

Reply:
Sure, updated the description with documentation. @MichaelSimons is that enough for your team to use this feature or is there anything else I should document?

Reply:
Thanks, the description is good.  It would be nice to capture this in a more centralized place long term.